src/classes/music.rb

```diff
@@ -0,0 +1,14 @@
+class MusicAlbum < Item
```

Comment:
Hi @JohnFTitor :smile:  you did a great job creating the MusicAlbum class using `TDD` :+1: , but I think you forgot to require the `Item` class in the first line.
When you run all the tests (rspec spec) the tests will pass, but when you test the individual file `music_spec.rb` it will fail because we need to require the `Item class` :smiley:

Reply:
Oh... I totally missed that. Thanks!